torch/jit/frontend.py

```diff
@@ -266,6 +266,12 @@ def build_Return(ctx, stmt):
         values = (stmt.value,) if not isinstance(stmt.value, ast.Tuple) else stmt.value.elts
         return Return(r, [build_expr(ctx, val) for val in values if val is not None])
 
+    @staticmethod
+    def build_Raise(ctx, stmt):
+        r = ctx.make_range(stmt.lineno, stmt.col_offset, stmt.col_offset + len("raise"))
+        expr = build_expr(ctx, stmt.exc)
```

Comment:
Python 2 doesn't have `exc` on `Raise` nodes ([docs](https://docs.python.org/2/library/ast.html#abstract-grammar)) so this throws an error. Probably best to figure how to do the same thing in Python 2 and put the functionality in `_six.py`